charts/scaffold/Chart.yaml

```diff
@@ -35,6 +35,10 @@ dependencies:
     version: 0.1.4
     repository: https://sigstore.github.io/helm-charts
     condition: tuf.enabled
+  - name: tsa
+    version: 0.1.0
+    repository: https://sigstore.github.io/helm-charts
+    condition: tuf.enabled
```

Comment:
This seems wrong, shouldn't this be `tsa.enabled`?

Reply:
Fixed!